docs/src/modules/components/MarkdownElement.js

```diff
@@ -2,474 +2,516 @@ import * as React from 'react';
-    fontFamily: brandingDarkTheme.typography.fontFamilyCode,
-    fontWeight: 400,
-    WebkitFontSmoothing: 'subpixel-antialiased',
-    // Reset for Safari
-    // https://github.com/necolas/normalize.css/blob/master/normalize.css#L102
-    fontSize: '1em',
-  },
-  // inline code block
-  '& :not(pre) > code': {
-    display: 'inline-block',
-    padding: '0 5px',
-    color: theme.palette.text.primary,
-    backgroundColor: alpha(theme.palette.primary.light, 0.15),
-    borderRadius: 5,
-    fontSize: theme.typography.pxToRem(13),
-    direction: 'ltr /*! @noflip */',
-  },
-  '& h1': {
-    ...theme.typography.h3,
-    fontSize: theme.typography.pxToRem(36),
-    fontFamily: `"PlusJakartaSans-ExtraBold", ${theme.typography.fontFamilySystem}`,
-    margin: '10px 0',
-    color: theme.palette.mode === 'dark' ? theme.palette.grey[50] : blueDark[900],
-    fontWeight: 800,
-  },
-  '& .description': {
-    ...theme.typography.subtitle1,
-    fontWeight: 400,
-    margin: '0 0 40px',
-  },
-  '& h2': {
-    ...theme.typography.h5,
-    fontFamily: theme.typography.fontFamilySystem,
-    fontWeight: 700,
-    color: theme.palette.mode === 'dark' ? theme.palette.grey[100] : theme.palette.grey[900],
-    margin: '40px 0 4px',
-  },
-  '& h3': {
-    ...theme.typography.h6,
-    fontFamily: theme.typography.fontFamilySystem,
-    fontWeight: theme.typography.fontWeightSemiBold,
-    color: theme.palette.mode === 'dark' ? theme.palette.grey[200] : theme.palette.grey[900],
-    margin: '24px 0 8px',
-  },
-  '& h4': {
-    ...theme.typography.subtitle1,
-    fontFamily: theme.typography.fontFamilySystem,
-    fontWeight: theme.typography.fontWeightSemiBold,
-    color: theme.palette.mode === 'dark' ? theme.palette.grey[300] : theme.palette.grey[900],
-    margin: '24px 0 8px',
-  },
-  '& h5': {
-    ...theme.typography.subtitle2,
-    fontWeight: theme.typography.fontWeightSemiBold,
-    color: theme.palette.mode === 'dark' ? theme.palette.grey[300] : theme.palette.grey[900],
-    margin: '20px 0 8px',
-  },
-  '& p, & ul, & ol': {
-    marginTop: 0,
-    marginBottom: 16,
-    color: theme.palette.mode === 'dark' ? theme.palette.grey[400] : theme.palette.grey[900],
-  },
-  '& ul': {
-    paddingLeft: 30,
-  },
-  '& h1, & h2, & h3, & h4': {
-    '& code': {
-      fontSize: 'inherit',
-      lineHeight: 'inherit',
-      // Remove scroll on small screens.
-      wordBreak: 'break-all',
-    },
-    '& .anchor-link-style': {
-      // To prevent the link to get the focus.
-      display: 'none',
-    },
-    '& a:not(.anchor-link-style):hover': {
-      color: 'currentColor',
-      borderBottom: '1px solid currentColor',
-      textDecoration: 'none',
+const Root = styled('div')(
+  ({ theme }) => ({
+    ...lightTheme.typography.body1,
+    color: `var(--muidocs-palette-text-primary, ${lightTheme.palette.text.primary})`,
+    '& strong': {
+      color: `var(--muidocs-palette-text-primary, ${lightTheme.palette.text.primary})`,
```

Comment:
Would this be better?

```suggestion
      color: `var(--muidocs-palette-text-primary, ${theme.colorScheme.light.palette.text.primary})`,

```

the coupling with the imports of `lightTheme` and `darkTheme` seems unhealthy for design consistency because the Material UI components can't use these values.